src/mysql/create.sql

```diff
@@ -0,0 +1,121 @@
+-- This file will create the database and its tables
```

Comment:
nit: Period at the end of this comment

Can you also add a link to your design doc for more context on the data design being used? 

```suggestion
-- This file will create the database and its tables.
--
-- For more context on the Mintern database design, see <design doc URL>.
```

Reply:
done